### ISSUE TITLE
The "mapProteinFeaturesToEvents" command keeps running!

### ISSUE THREAD

**Issue by nandan75** [DiogoVeiga/maser#24]
Hi Diogo,

I have been using "maser" to map my events to specific transcripts and I like the module. It is very useful.
I recently discovered the "mapping to protein" feature of the module. 
Very excited.

**I was trying the samples in the tutorial at**
https://www.bioconductor.org/packages/release/bioc/vignettes/maser/inst/doc/Protein_mapping.html#3_session_info

But could not reach the plotting step.

When I tried to run the commands
ripk2_events <- geneEvents(hypoxia_filt, "RIPK2")
ripk2_mapped <- mapTranscriptsToEvents(ripk2_events, ens_gtf)
ripk2_annot <- mapProteinFeaturesToEvents(ripk2_mapped, 
                                          tracks = c("Domain_and_Sites"), 
                                          by = "category")

The command **"mapProteinFeaturesToEvents"** kept running! never reached completion !
I have attached a screen shot to show what I mean.

Thanks in advance .

regards ,

Nandan
<img width="1548" alt="Screen Shot 2020-06-10 at 12 02 10 pm" src="https://user-images.githubusercontent.com/12062345/84219037-041d9f80-ab13-11ea-848b-2ff207dad852.png">

**Comment by nandan75** [comment]
Hi Doigo,

I just got an error, a minute after I opened this issue. 
Attaching it for your reference.

regards,

Nandan

<img width="1497" alt="Screen Shot 2020-06-10 at 12 09 30 pm" src="https://user-images.githubusercontent.com/12062345/84219230-64acdc80-ab13-11ea-82df-767ca8757ade.png">

**Comment by nandan75** [comment]
Aplogise for multiple posts.

I re-ran the last step and I got it executed the next time. 

So assuming that the problem could be possibly due to slow internet connectivity (since I found that the "ftp path" in the above error for the bed files - does exists), I ran the script on my university machine (with fast speed internet). 
It still took a while to run and eventually finished.

So I am assuming the last step does take a while ? Has anyone else got a similar feedback?

regards,

Nandan
